meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-4x2g-kf-hdmi-domd.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the H3ULCB Kingfisher board
+ * running XEN hypervisor
+ *
+ * Copyright (C) 2016-2018 Renesas Electronics Corp.
+ * Copyright (C) 2016 Cogent Embedded, Inc.
+ */
+
+#include "r8a77951-h3ulcb-4x2g-kf-domd.dtsi"
+
+/ {
+	rvgc {
+		displays {
+			display-0 {
+				display-map = <1>;
```

Comment:
But this should be changed to
display-map = <0x3>;
and resolution to
1280x720